themes/classic/templates/_partials/head.tpl

```diff
@@ -43,10 +43,22 @@
     <link rel="canonical" href="{$page.canonical}">
   {/if}
   {block name='head_hreflang'}
-      {foreach from=$urls.alternative_langs item=pageUrl key=code}
-            <link rel="alternate" href="{$pageUrl}" hreflang="{$code}">
-      {/foreach}
+    {foreach from=$urls.alternative_langs item=pageUrl key=code}
+      <link rel="alternate" href="{$pageUrl}" hreflang="{$code}">
+    {/foreach}
   {/block}
+  
+  {include file="_partials/microdata/head-jsonld.tpl"}
```

Comment:
I'd put it into

`{block name='jsonld'}`

Reply:
{block name='head_microdata'} yep, to keep consistency with other head block in head.tpl (done)